lightning/src/ln/channelmanager.rs

```diff
@@ -668,7 +668,7 @@ impl HTLCSource {
 	pub fn dummy() -> Self {
 		assert!(cfg!(not(feature = "grind_signatures")));
 		HTLCSource::OutboundRoute {
-			path: Path { hops: Vec::new(), blinded_tail: None },
+			path: Path { hops: Vec::new(), trampoline_hops: Vec::new(), blinded_tail: None },
```

Comment:
nit: every other vec in this patch is initialized with `vec![]`, perhaps make this consistent ? your call feel free to dismiss.